src/plugins/plugin-build-script.ts

```diff
@@ -10,16 +12,19 @@ export function buildScriptPlugin({
   input: string[];
   output: string[];
   cmd: string;
-}) {
+}): SnowpackPlugin {
```

Comment:
Is there a better type we could use for the function params?

Reply:
+1, moved to match plugin interface to make this easier to pull out